media_driver/linux/common/codec/ddi/media_ddi_encode_vp9.cpp

```diff
@@ -1062,7 +1062,12 @@ VAStatus DdiEncodeVp9::ReportExtraStatus(
     codedBufStatus->next_frame_width = encodeStatusReport->NextFrameWidthMinus1 + 1;
     codedBufStatus->next_frame_height = encodeStatusReport->NextFrameHeightMinus1 + 1;
 
-    codedBufferSegment->next = codedBufStatus;
+    /*
+     * Ignore the private status buffer temporarily. According to the comment for VACodedBufferVP9Status in VA-API,
+     * driver must set codedBufferSegment->status to be VA_CODED_BUF_STATUS_CODEC_SPECIFIC, however
+     * VA_CODED_BUF_STATUS_CODEC_SPECIFIC is not defined in VA-API
```

Comment:
This sounds like a bug in VAAPI spec. Is there libva bug for that? If not, can you, please, submit the one and link it here?

Reply:
@dvrogozh It seems no one is interested in the private status buffer.

Reply:
@XinfengZhang @dvrogozh I think it would be better to ignore this private buffer as no one is interested in this buffer. We may define VA_CODED_BUF_STATUS_CODEC_SPECIFIC in VAAPI spec and add this buffer back later if someone want to use this private buffer. How do you think?

Reply:
Hi All,
Any update on merge status for this commit in media driver or we are looking for fix in gstreamer-vaapi?